stats.rb

```diff
@@ -134,17 +138,19 @@ def tablecreation(title,pr_array)
 html.push("<html><title>PRs that Require Triage</title>")
 html.push("<h1>PRs that Require Triage</h1>")
 
-htmlchunk = tablecreation("PRs that are uncommented:",array_uncommented_pulls)
+htmlchunk = tablecreation("PRs that are have 0 comments:",array_uncommented_pulls)
 html.push(htmlchunk)
-htmlchunk = tablecreation("PRs that have been dormant for 15 days:",array_needs_prompt_pulls)
+htmlchunk = tablecreation("Last comment from puppet, no response for 15 days (needs ping):",array_needs_prompt_pulls)
 html.push(htmlchunk)
-htmlchunk = tablecreation("PRs that have been dormant for 30 days:",array_last_comment_pulls)
+htmlchunk = tablecreation("Last comment from puppet, no response for 30 days (needs closed):",array_last_comment_pulls)
 html.push(htmlchunk)
 htmlchunk = tablecreation("PRs that have yet to be commented on by a puppet member:",array_puppet_uncommented_pulls)
 html.push(htmlchunk)
-htmlchunk = tablecreation("PRs that community have asked for help:",array_mentioned_pulls)
+htmlchunk = tablecreation("PRs that community have asked for help (mentioned a puppet member):",array_mentioned_pulls)
+html.push(htmlchunk)
+htmlchunk = tablecreation("PRs that require rebase (needs comment and a label):",array_needs_rebase_pulls)
 html.push(htmlchunk)
-htmlchunk = tablecreation("PRs that require rebase:",array_needs_rebase_pulls)
+htmlchunk = tablecreation("PRs that should die, no activity for 40 days:",array_needs_rebase_pulls)
```

Comment:
Can this be reworded to 'should be closed' or something less harsh than 'should die'?